### ISSUE TITLE
plugin web

### ISSUE THREAD

**Issue by formu25400** [MarlinFirmware/Marlin#12797]
Hello
And is it plausible on marlin 2.0 to have the mini web interface as on smoothiewar
On my MKS SBASE v1.3 card

![capture](https://user-images.githubusercontent.com/36988342/50597660-76af3800-0ea8-11e9-9e0c-6e95553d7c8b.JPG)

**Comment by gloomyandy** [comment]
No, at least not at the moment, none of the network functionality of the MKS Sbase has been enabled.

**Comment by weed2all** [comment]
You can do this with an esp8266...take a look on the link
https://youtu.be/pJGBRriNc9I

**Comment by boelle** [comment]
@formu25 did @weed2all suggestion help you?

maybe reword the title line to: [FEATURE REQUEST] Enable network functionality on MKS Sbase

**Comment by boelle** [comment]
@thinkyhead i think we can close this one
